### PR TITLE
#314 fix Excluse ics & inline images on attachment sidebar panel

### PR DESCRIPTION
**Pull request by fabienmoyon**
Capture d’écran de 2021-03-09 17-50-25

### CODE REVIEW

**Review by alagane — APPROVED** [review]

**Comment by chibenwa** [comment]
@tuanlc would this qualify for a review?

File: src/linagora.esn.unifiedinbox/app/providers.js [hunk 1]
```diff
@@ -40,9 +40,8 @@ require('./directives/lists.js');
 
                         if (trueAttachment.length > 0) {
                           message.attachments = trueAttachment;
+                          messageTrueAttachmentList.push(message);
                         }
-
-                        messageTrueAttachmentList.push(message);
```
Comment (tuanlc) [inline src/linagora.esn.unifiedinbox/app/providers.js:45]: IMO, you should modify existing tests or writes new test cases for this change?

**Review by tuanlc — COMMENTED** [review]
Your commits are strange with 2 first commits seem useless.

**Comment by tanandy** [comment]
up

**Review by alagane — APPROVED** [review]

**Review by oubchid — APPROVED** [review]

**Pull request merged**